app/views/standard/fileChecksResults.scala.html

```diff
@@ -28,7 +28,7 @@ <h3 class="govuk-notification-banner__heading">
                     Now that your records have been uploaded you can proceed with the transfer. In the next step you will be given the opportunity to add metadata to your records before transferring them.
                 </strong>
             </div>
-            <a class="govuk-button" href="@routes.AdditionalMetadataController.start(consignmentId)" role="button" draggable="false" data-module="govuk-button">
+            <a class="govuk-button" href="@routes.AdditionalMetadataEntryMethodController.additionalMetadataEntryMethodPage(consignmentId)" role="button" draggable="false" data-module="govuk-button">
```

Comment:
This will need to be wrapped with the FAB as we don't want the new re-direct released on production until its ready.